rfc-drafts/stream.md

```diff
@@ -221,31 +231,142 @@ existing code, perhaps as part of an edition migration.
 
 Designing such a migration feature is out of scope for this RFC.
 
-## IntoStream / FromStream traits, mirroring iterators
+## IntoStream / FromStream traits
+
+The `IntoStream` trait would provide a way to convert something into a `Stream`.
+
+The `FromStream` trait would provide a way to convert a `Stream` into something else.
+
+Implementing the `IntoStream` and `FromStream` traits on `Stream` is currently blocked on needing [`async fn` available in traits]( http://smallcultfollowing.com/babysteps/blog/2019/10/26/async-fn-in-traits-are-hard).
```

Comment:
...that would then clarify why this is true (in fact, I'm not entirely convinced this is true)